.github/workflows/pr.yml

```diff
@@ -138,6 +138,7 @@ jobs:
           chmod a+x minikube-*
           sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
           sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
+          KUBECONFIG=$(pwd)/testhome/kubeconfig MINIKUBE_HOME=$(pwd)/testhome ./minikube-linux-amd64 delete --all --purge
```

Comment:
do we really need KUBECONFIG for `minikube delete`?

Reply:
I don't think so! Copy-pasted from the testing line :/ Good catch! Fixed